### ISSUE TITLE
Unable to Play Music

### ISSUE THREAD

**Issue by collinjackson93** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#1459]
**OS:** Windows 10 Version 1607

**Issue Descriptions:** The application loads fine and I am able to browse through all pages (Music Library, Browse Stations, etc) without issue, but when I try to play anything (song, playlist, station, etc), it doesn't start playing. If I click play on a station, the loading indicator appears, but then it does not start playing. I have already tried restarting my computer, to no avail.

I know this isn't much information to go off of, but I'm not too sure where to start looking for debug info.

**Comment by heru-ra** [comment]
Same issue, on Debian Linux.

Yesterday the UI would load and be interactive, but no stations would play. Today the UI won't even load, I just get a loading "spinner" graphic the whole time.

**Comment by collinjackson93** [comment]
Okay, I'm glad I'm not the only one. I didn't try using the application yesterday, but the day before (08/10) it worked perfectly fine. I wonder if maybe there was an API update?

**Comment by jason-mehmel** [comment]
Having a similar issue. Can't browse music though, I'm just getting the circular loading graphic.

**Comment by joshkeating** [comment]
I have a similar issue as well, on launch I only get the loading spinner.

Arch Linux 4.6

**Comment by Loran425** [comment]
Also on Ubuntu 16.04, my problem seemed to be out of date software for me. 

Try this on Ubuntu to check version.
`dpkg -l google-play-music-desktop-player` 

My working version shows 3.4.3-1 as my version

**Comment by MarshallOfSound** [comment]
Guys, please check the closed issues before raising new ones...

Duplice #1454 #1453 #1445

**Comment by MarshallOfSound** [comment]
TLDR Update to version `3.4.3`, windows and macOS users should receive the update automatically.  Linux users have to do linux things and update automatically
